core/templates/dev/head/css/oppia.css

```diff
@@ -680,10 +680,17 @@ textarea {
   height: 198px;
 }
 
+.oppia-footer-margin{
```

Comment:
No, you don't need to add an empty css declaration for media queries to work. When the desired size is hit (in our case, if the width is less than or 768px as its maximum width), the code _inside_ the media query will run. It doesn't need to look up anywhere else in the file.